Lib/pdb.py

```diff
@@ -1509,6 +1510,12 @@ def _msg_val_func(self, arg, func):
         except:
             self._error_exc()
 
+    def _safe_repr(self, obj, expr):
+        try:
+            return repr(obj)
+        except Exception as e:
+            return _rstr(f"*** repr({expr}) failed: {self._format_exc(e)} ***")
```

Comment:
Do we use f-strings or ``%`` in this script? Or a mixture of the two?

Reply:
I'm using f-strings for all new code. There are plenty of usages with `%` as `pdb` is old. Do you think it would be helpful to have a PR to update all strings to f-strings? It would eliminate the mixture.

Reply:
Probably not worth the risk of breaking something.